vaccine/data/suburbs.py

```diff
@@ -110,7 +110,7 @@ async def ussd_search(self, province_id, search_text, municipality_id=None):
             ]
             return (True, self._filter_duplicate_municipalities(municipalities))
         else:
-            return (False, [(id, name) for name, _, id in possibilities[:3]])
+            return (False, [(id, name) for name, _, id in possibilities])
```

Comment:
I think we'll need to do this now in the state, as the logic is still `if len(possibilities) > 3` in here, where maybe we should do `if len(possibilities) > len(possibilities_that_fit)` in the state. But also happy just to keep it at 3, then you can ignore this comment.

Reply:
Happy to leave it at 3 for now 😃